java/jdbc/ConnectionSamples/JDBCDBTokenSample.java

```diff
@@ -0,0 +1,98 @@
+
+/* Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.*/
+/*
+   DESCRIPTION    
+   The code sample shows how to use the JDBC driver to establish a connection
+   to the Autonomous Database (ADB) using database token 
```

Comment:
If "database token" is the term we use in our documentation, this is fine. But to me, I think "access token" would be more accurate. It is common to find the term "access token" in literature about token based authentication systems.